internal/plog/config_test.go

```diff
@@ -230,11 +233,11 @@ testing.tRunner
 	require.True(t, scanner.Scan())
 	require.NoError(t, scanner.Err())
 	require.Equal(t, fmt.Sprintf(`I1121 23:37:26.953313%8d config_test.go:%d] "created before mode change: warn" is="old" i am="old1" warning=true`,
-		pid, startLogLine+2+13+14+11+12+24+28+6+26+6+6+7+1+10+9+6), scanner.Text())
+		pid, getLineNumberOfCaller()-5), scanner.Text())
```

Comment:
lolz, these series of numbers from before your refactor :)

Reply:
It was too difficult to figure out, so I changed it.